src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -37,6 +37,7 @@ limitations under the License.
         <tag name="NOT" description="Notification" />
         <tag name="YDSCH" description="Year Day Access Schedules" />
         <tag name="HDSCH" description="Holiday Schedules" />
+        <tag name="UBOLT" description="Unbolting" />
```

Comment:
Can you please split out the XML + general codegen changes from the implementation changes?  That will make this much saner to review, and will let us get the XML in and unblock testing more quickly, even as we continue to sort out the implementation.